docs/topics/Threads.md

```diff
@@ -24,9 +24,9 @@ The list of gateway events that may be dropped includes, but is not limited to:
 - THREAD_MEMBER_UPDATE
 - THREAD_MEMBERS_UPDATE
 
-## New Thread Fields
+## Thread Fields
 
-Since threads are a new [type of channel](#DOCS_RESOURCES_CHANNEL/channel-object-channel-types), they share and re-purpose a number of the existing fields on a [channel](#DOCS_RESOURCES_CHANNEL/channel-object) object:
+Since threads are a new [type of channel](#DOCS_RESOURCES_CHANNEL/channel-object-channel-types), they share and repurpose a number of the existing fields on a [channel](#DOCS_RESOURCES_CHANNEL/channel-object) object:
```

Comment:
if "new" is remove from the header, shouldn't it be remove here too?

Reply:
just updated this (except in the forums section since that can be folded into #5494)